ChangeLog.md

```diff
@@ -20,7 +20,12 @@ See docs/process.md for more on how version tagging works.
 
 Current Trunk
 -------------
-- values returns from `pathconf` now match the defintions found in header files
+- `DISABLE_EXCEPTION_CATCHING=2` is now deprecated since it can be inferred from
+  the presence of the `EXCEPTION_CATCHING_ALLOWED` list.  This makes
+  `DISABLE_EXCEPTION_CATCHING` a simple binary option (0 or 1) which defaults to
+  0 which will be set to 1 internally if `EXCEPTION_CATCHING_ALLOWED` list is
+  specified.
+- Values returned from `pathconf` now match the definitions found in header files
```

Comment:
Is this supposed to be in another PR?

Reply:
I just fixed a couple of typos in that line here while I was adding the new entry.

Reply:
Right it was an existing line. I missed it, sorry.